docs/Tutorial_Consensus_Motif.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>I've deposited a cleaner version of the CSV files. Can you please replace this code block with:</p><p> </p><pre class="ql-syntax" spellcheck="false">snt_oi = [6, 7, 9, 10, 16, 24]
eye = [None] * len(snt_oi)
fs = 50 # eog signal was downsampled to 50 Hz
&nbsp;
for i, s in enumerate(snt_oi):
&nbsp; &nbsp; eye[i] = pd.read_csv(f'https://zenodo.org/record/4288978/files/EOG_001_01_{s:03d}.csv?download=1').iloc[:, 0].values
&nbsp; &nbsp; &nbsp; &nbsp;&nbsp;
# the literal sentences
sentences_df = pd.read_csv(f'https://zenodo.org/record/4288978/files/test_sent.jp.csv?download=1', index_col=0)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Thanks!</p>